src/main/java/io/appium/mitmproxy/MitmproxyJava.java

```diff
@@ -9,27 +10,36 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.InetSocketAddress;
+import java.net.ServerSocket;
 import java.net.Socket;
-import java.net.URISyntaxException;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeoutException;
 import java.util.function.Function;
 
 public class MitmproxyJava {
 
     private String mitmproxyPath;
+    private Function<InterceptedMessage, InterceptedMessage> messageInterceptor;
```

Comment:
final ?

Reply:
same for properties below that are initializaed in the constructor

Reply:
ok, will change